### ISSUE TITLE
Error: 'overlap.xypolygon' is not an exported object from 'namespace:spatstat'

### ISSUE THREAD

**Issue by palomapali** [AndrewLJackson/SIBER#24]
I got this error: 'overlap.xypolygon' is not an exported object from 'namespace:spatstat'
When trying to use the overlap function. 

Everything is up to date and I can't figure it out what is going on.

**Comment by AndrewLJackson** [comment]
hi

can you please check that you are using the latest development version of SIBER (not from cran) available from 
# install.packages("devtools") # install if necessary
  devtools::install_github("andrewljackson/SIBER@master",
  build_vingettes = TRUE)
  library(SIBER)

A change to the dependent package spatstat meant i had to implement a quick hotfix that is not yet available on the cran version

**Comment by palomapali** [comment]
Hi Andrew,
Thanks for the quick reply!
That was the problem.
Thanks! Working now!

**Comment by vanmieva** [comment]
Hello! I am new to GitHub, so I am not sure if this is posted somewhere else. However, I am currently following the vignette at https://github.com/AndrewLJackson/SIAR-examples-and-queries/blob/master/learning-resources/siber-comparing-populations.Rmd and trying to use the overlap function but I am getting the same error as palomapali described above. 

I have updated SIBER and all the spatstat packages as well. I also ran install.packages("devtools")  but I am still getting the error: 

Error in overlap.xypolygon(e1, e2) : 
  could not find function "overlap.xypolygon"

I am wondering if the overlap.xypolygon function has been retired perhaps?  Any help would be greatly appreciated!!

Kindest, 
Victoria
